src/EditorFeatures/Core/Implementation/IntelliSense/Completion/Controller.cs

```diff
@@ -66,8 +68,20 @@ internal partial class Controller :
             _undoHistoryRegistry = undoHistoryRegistry;
             _allCompletionProviders = allCompletionProviders;
             _autoBraceCompletionChars = autoBraceCompletionChars;
-            _isDebugger = isDebugger;
-            _isImmediateWindow = isImmediateWindow;
+
+            var tagsBuilder = ImmutableArray.CreateBuilder<string>();
+
+            if (isDebugger)
```

Comment:
If you Eventually™ expect this type becomes public, we'll probably need the controller to just have a IEnumerable<string> that comes in. It seems very odd that the two booleans are called out, especially since I think there's only caller of this.